src/pysplunkhec.py

```diff
@@ -0,0 +1,89 @@
+# the following allows JSON to be posted to Splunk HEC (HTTP Event Collector)
+
+import os
+import requests
+import json
+from src.kms import decrypt
+
+class PySplunkHEC:
+    def __init__(
+        self,
+        hec_token,
+        hec_host,
+        index="main",
+        host="null",
+        source="null",
+        sourcetype="json",
+        hec_port='443'
+        ):
+        self.index = index
+        self.host = host
+        self.source = source
+        self.sourcetype = sourcetype
+        self.hec_token = hec_token
+        self.uri = "https://" + hec_host + ":" + hec_port + "/services/collector"
+
+    def base_json_event(self):
+        return {
+            "host": self.host,
+            "source": self.source,
+            "sourcetype": self.sourcetype,
+            "index": self.index,
+            "event": {},
+            }
+
+    def send(self, payload):
+        headers = {
+            'Authorization': 'Splunk ' + self.hec_token,
```

Comment:
`f'Splunk {self.hec_token}'`

The token has a space in it?

Reply:
changed as requested, yes there is a space between Splunk and the token